src/Pages/Detail/Detail.js

```diff
@@ -1,8 +1,132 @@
 import React, { Component } from "react";
+import DetailLeft from "./DetailLeft";
+import DetailRight from "./DetailRight";
+import "./Detail.scss";
 
 class Detail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      images: [],
+      quantity: 1,
+      maxQuantity: 0,
+      sizes: [],
+      title: "",
+      categories: "",
+      price: 0,
+      options: [],
+      disabled: false,
+    };
+  }
+
+  doRequest = () => {
+    fetch(
+      `http://10.58.1.235:8000/product/${this.props.match.params.productId}`,
+      {
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+      .then((res) => res.json())
+      .then((res) => {
+        console.log(res);
+        const {
+          images,
+          title,
+          categories,
+          price,
+          sizes,
+          options,
+        } = res.product;
+        this.setState({
+          images: images,
+          title: title,
+          categories: categories,
+          price: price,
+          sizes: sizes,
+          options: options,
+          disabled: false,
```

Comment:
```suggestion
        this.setState({
          images,
          title,
          categories,
          price,
          sizes,
          options,
          disabled: false,
```
객체 단축 속성명을 활용하면 이렇게 줄여줄 수 있습니다. 객체의 키값이랑 value의 값이 같으면 줄여줄 수 있습니다